previous-versions/2019.md

```diff
@@ -32,10 +32,10 @@ This talks about both a practise of being able to spin up your application ecosy
 If your tests pass you should never be afraid to release to production; moving fast, rely on your tests, logging and knowledge that you can always roll back.
 
 - process as an enabler not a restriction
-- using the same artifact in all environments
-- make it easy to release often
 - logging as part of a feature
 - fix fast
+- **new:** invest time in making it as easy as possible to iterate
+- **new:** don't be rushed
```

Comment:
+1 to don't **be** rushed

Reply:
Don't _be_ rushed because this suggests being made to rush by external actors. The ideal is that nobody puts developers under pressure to finish something or get something out. 
I think the "fix fast" doesn't relate to the speed you code at, its more specifically to spotting and reacting to issues quickly, which requires good monitoring, and then being able to put a fix out, which requires good processes like ci and automated testing.